src/AirSpeedSensor.cc

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (C) 2023 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#if defined(_MSC_VER)
+  #pragma warning(push)
+  #pragma warning(disable: 4005)
+  #pragma warning(disable: 4251)
+#endif
+#include <gz/msgs/air_speed_sensor.pb.h>
+#if defined(_MSC_VER)
+  #pragma warning(pop)
+#endif
+#include <gz/msgs/Utility.hh>
+
+#include <gz/common/Profiler.hh>
+#include <gz/transport/Node.hh>
+
+#include "gz/sensors/GaussianNoiseModel.hh"
+#include "gz/sensors/Noise.hh"
+#include "gz/sensors/SensorTypes.hh"
+#include "gz/sensors/SensorFactory.hh"
+#include "gz/sensors/AirSpeedSensor.hh"
+
+using namespace gz;
+using namespace sensors;
+
+static constexpr double kPressureOneAtmospherePascals = 101325.0;
+// static constexpr double kSeaLevelTempKelvin = 288.15;
+
+static constexpr auto DEFAULT_HOME_ALT_AMSL = 0.0f; // altitude AMSL see level [m]
```

Comment:
I see this is used for addition below but since it's zero is it needed?

Reply:
maybe in the future we can add a param to ajust this variable